index.js

```diff
@@ -108,7 +107,6 @@ module.exports = {
     ],
     'scss/double-slash-comment-whitespace-inside': 'always',
     'scss/declaration-nested-properties-no-divided-groups': true,
-    'scss/operator-no-newline-after': true,
```

Comment:
This rule conflicts with Prettier, so removing it (at least, was creating issues when tested in Backpack).